acp/dac/local_test.go

```diff
@@ -68,32 +68,22 @@ func (m *mockInvalidIdentity) NewToken(duration time.Duration, audience immutabl
 
 var invalidIdentity = &mockInvalidIdentity{}
 
-var validPolicyID string = "196f584c831c596489278ed2c98ee7a42fc2cf85b2e86276165a03b53b4684fa"
+var validPolicyID string = "177c8aecb1656155806f79b09cfc2c922b82872ed313a0263a6040d80253bdf0"
 var validPolicy string = `
-name: test
 description: a policy
-
-actor:
-  name: actor
-
+name: test
 resources:
-  users:
-    permissions:
-      read:
-        expr: owner + reader
-      update:
-        expr: owner
-      delete:
-        expr: owner
-
-    relations:
-      owner:
-        types:
-          - actor
-      reader:
-        types:
-          - actor
- `
+- name: users
+  permissions:
+  - name: delete
+  - expr: reader
```

Comment:
nitpick: Slight preference to have name `first` the `expr`:

```suggestion
  - name: read
    expr: reader
```

Ignore if hard to do with your migrating script.